app/icons/Bed.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { forwardRef } from 'react'
+import type { SVGProps } from 'react'
+
+export default forwardRef<SVGSVGElement, SVGProps<SVGSVGElement>>(function Bed(props, ref) {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor"
```

Comment:
remove the `xmlsns="..."`; it isn't needed
also `props` and `ref` are currently unused, check out https://github.com/helpwave/web/blob/main/app/icons/Check.tsx maybe